docs/modules/sql/pages/user-defined-types.adoc

```diff
@@ -620,9 +505,12 @@ Using `organization` from the example above.
 hz.getSql().execute("UPDATE users SET organization = ?", organization);
 ----
 
-Updating nested UDT column with Query Parameter:
+Updating nested UDT column with query parameter:
 
 [source,java]
 ----
 hz.getSql().execute("UPDATE users SET organization = ((organization).id, (organization).name, ?)", office);
-----
\ No newline at end of file
+----
+
+== Rolling Upgrade Notes
```

Comment:
this affects also normal upgrade with persistent SQL catalog. both are EE features - we should have "enterprise" marker here

Reply:
fef8c8c2